src/components/common/Transactions/Transactions.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import {
+  CircularProgress, Divider, Paper, RaisedButton, Table, TableBody, TableFooter, TableHeader, TableHeaderColumn,
+  TableRow,
+  TableRowColumn
+} from 'material-ui'
+import globalStyles from '../../../styles'
+import styles from './styles'
+import { getScannerById } from '../../../network/settings'
+import { Translate } from 'react-redux-i18n'
+
+const mapStateToProps = (state) => ({
+  selectedNetworkId: state.get('network').selectedNetworkId,
+  selectedProviderId: state.get('network').selectedProviderId
+})
+
+@connect(mapStateToProps, null)
+class Transactions extends Component {
+  getEtherscanUrl = () => {
+    const {selectedNetworkId, selectedProviderId} = this.props
+    const baseScannerUrl = getScannerById(selectedNetworkId, selectedProviderId)
+    return baseScannerUrl ? `${baseScannerUrl}/tx/` : null
+  }
+
+  render () {
+    const etherscanHref = this.getEtherscanUrl()
+    const {transactions, isFetching, toBlock} = this.props
+
+    return (
+      <Paper style={globalStyles.paper} zDepth={1} rounded={false}>
+        <h3 style={globalStyles.title}><Translate value='tx.transactions' /></h3>
+        <Divider style={{backgroundColor: globalStyles.title.color}} />
+        <Table selectable={false}>
+          <TableHeader adjustForCheckbox={false} displaySelectAll={false}>
+            <TableRow>
+              <TableHeaderColumn style={styles.columns.id}><Translate value='terms.block' /></TableHeaderColumn>
+              <TableHeaderColumn style={styles.columns.hash}><Translate value='terms.hash' /></TableHeaderColumn>
+              <TableHeaderColumn style={styles.columns.time}><Translate value='terms.time' /></TableHeaderColumn>
+              <TableHeaderColumn style={styles.columns.actions}><Translate value='terms.action' /></TableHeaderColumn>
```

Comment:
Its header and here single form like time, hash and block. 
'styles.colums.actions' renamed